Lesson Plans/Flight Instructor/CFI/CFI1.1 Human Behavior.md

```diff
@@ -0,0 +1,126 @@
+				1. Some has fundamental interest in aviation and just love flying
+				2. Some see aviation as an opportunity to develop technical, physical, communication, problem-solving abilities.
+				3. Some see this as a way to boost their ego.
+				4. Some are following a family's footstep or promising career.
+			2. *They all offer some type of reward in exchange for the hard work.*
+		2. Rewarding Success
+			1. Provide frequent and on-the-moment positive feedback during training
+			2. Relating their accomplishments to lesson objectives
+			3. Commenting favorably on their progress and level ability (If it's close, at, or even exceeding ACS standard, we can also point it out to them.)
+		3. Presenting New Challenges
+			1. *When the learner can perform a skill consistently to ACS / PTS requirements*, we should challenge them to continue to improve it so it can be demonstrated under pressure or when distracted.
+			2. You can also add-on new problems or situations to challenge the student.
+		4. Drops in Motivation
+			1. *It is natural for motivation to wane after the initial excitement wear off*, especially after major training events such as solo, phase checks or practice tests.
+			2. Learners might come unprepared or give out the vibe that aviation is no longer a priority.
+			3. It is often helpful to remind them *their own stated goals* for seeking aviation training.
+			4. Learning plateaus are a common source of frustration, discouragement — They should be encouraged to continue to work hard and be reassured that *it will pass and that the results will follow.*
+4. Human Needs
+	1. ![[Hierarchy of Needs.png]]*Maslow's Hierarchy of Needs* until the needs are satisfied, one cannot focus on learning.
+	2. **Physiological**: Instructors should monitor the learners to make sure their basic physical needs are met (not hungry, dehydrated, tired, etc.)
+	3. **Security**: Instructor should be aware of the learner's fear of flying and ease them into it carefully.
+	4. **Belonging**: Instructor should welcome the learners into the field of aviation, be it a career or hobby for learners.
+	5. **Esteem**: Self-esteem can come from internal judgement but most people seek external esteem through social approval (status, recognition, respect, etc.) It might not only be a strong influence on the instructor-learner relationship, it may be the main reason for a learner's interest in aviation training.
+	6. **Cognitive and Aesthetic**: *This layer is added in the later years and thus not in the picture shown above.* Cognitive is the need to know and understand, to control and make informed decisions. Aesthetic are emotional needs, if an instructor does not like a learner, that subtle feeling may leak into their teaching.
+	7. **Self-Actualization**: Realizing one's own potential for continued development / personal goals.
+	8. *Help learners satisfy their needs in a manner that creates a healthy learning environment.*
+5. Defense Mechanisms
```

Comment:
Could make this into a table with examples? Or maybe not, but might be nice to break it up somehow